source/PyMaterialX/PyMaterialXCore/PyValue.cpp

```diff
@@ -39,4 +43,20 @@ void bindPyValue(py::module& mod)
     BIND_TYPE_INSTANCE(booleanarray, mx::BoolVec)
     BIND_TYPE_INSTANCE(floatarray, mx::FloatVec)
     BIND_TYPE_INSTANCE(stringarray, mx::StringVec)
+
+    mod.attr("TypedValue_boolean").doc() = "A `Value` object that stores a value of type `bool`.";
+    mod.attr("TypedValue_booleanarray").doc() = "A `Value` object that stores a `list` of `bool` values.";
+    mod.attr("TypedValue_color3").doc() = "A `Value` object that stores a value of type `Color3`.";
+    mod.attr("TypedValue_color4").doc() = "A `Value` object that stores a value of type `Color4`.";
+    mod.attr("TypedValue_float").doc() = "A `Value` object that stores a value of type `float`.";
+    mod.attr("TypedValue_floatarray").doc() = "A `Value` object that stores a `list` of `float` values.";
+    mod.attr("TypedValue_integer").doc() = "A `Value` object that stores a value of type `int`.";
+    mod.attr("TypedValue_integerarray").doc() = "A `Value` object that stores a `list` of `int` values.";
+    mod.attr("TypedValue_matrix33").doc() = "A `Value` object that stores a value of type `Matrix33`.";
+    mod.attr("TypedValue_matrix44").doc() = "A `Value` object that stores a value of type `Matrix44`.";
+    mod.attr("TypedValue_string").doc() = "A `Value` object that stores a value of type `str`.";
+    mod.attr("TypedValue_stringarray").doc() = "A `Value` object that stores a `list` of `str` values.";
+    mod.attr("TypedValue_vector2").doc() = "A `Value` object that stores a value of type `Vector2`.";
+    mod.attr("TypedValue_vector3").doc() = "A `Value` object that stores a value of type `Vector3`.";
+    mod.attr("TypedValue_vector4").doc() = "A `Value` object that stores a value of type `Vector4`.";
```

Comment:
We may want to simplify these, e.g.
```diff
-    mod.attr("TypedValue_boolean").doc() = "A `Value` object that stores a value of type `bool`.";
+    mod.attr("TypedValue_boolean").doc() = "A `Value` object that stores a `bool` value.";
```
Or even more radical:
```diff
-    mod.attr("TypedValue_boolean").doc() = "A `Value` object that stores a value of type `bool`.";
+    mod.attr("TypedValue_boolean").doc() = "A `bool` value.";
```

Reply:
Simplified in https://github.com/AcademySoftwareFoundation/MaterialX/pull/2051/commits/056a7efcd5a276074dd9871f6e922c806f2a6291